### ISSUE TITLE
배포 Model Lab과 공식 최종 모델의 조합·선정 기준 불일치

### ISSUE THREAD

**Issue by GoonGuMa** [devlee328288/Alpha_Stack#280]
## 확인된 문제

현재 배포된 대시보드의 Model Lab에서 표시하는 `BEST MODEL`이 발표 및 최종 결과에서 확정한 공식 모델과 일치하지 않습니다.

Model Lab을 실행했을 때 LightGBM이 `BEST MODEL`로 표시되지만, 확인 결과 Model Lab은 공식 최종 모델 선정과 다른 피처 조합 및 선정 기준을 사용하고 있습니다.

### 1. 피처 조합 E 고정

`dashboard/services/model_service.py`에 다음과 같이 설정돼 있습니다.

```python
DEFAULT_COMBINATION = "E"
```

Model Lab에서 `run_all_models()`를 호출할 때 `combination`을 별도로 전달하지 않으므로, 실제 모델 학습과 비교에도 기본값인 조합 E가 사용됩니다.

화면 상단에도 `COMBINATION E`가 표시되며, 현재 조합을 변경하는 UI는 없습니다.

### 2. 비교 모델

Model Lab은 다음 네 모델을 동일한 조합 E로 실행합니다.

- LogisticRegression
- RandomForest
- XGBoost
- LightGBM

### 3. 조화평균으로 BEST MODEL 선정

Model Lab은 다음 세 지표의 조화평균인 `core_harmonic_mean`을 기준으로 결과를 내림차순 정렬합니다.

- Accuracy
- Macro F1
- 하락 Recall

이 조화평균이 가장 높은 모델을 화면에서 `BEST MODEL`로 표시하기 때문에, 실행 결과에 따라 LightGBM이 BEST로 표시됩니다.

그러나 해당 조화평균은 앞선 이슈 논의에서 공식 최종 모델 선정 기준으로 사용하지 않기로 정리한 지표입니다.

## 공식 최종 모델

발표자료와 최종 홀드아웃 및 마지막 `T+1~T+6` 실행에서 사용한 모델은 다음과 같습니다.

| 트랙 | 피처 조합 | 최종 모델 |
|---|---|---|
| 개별종목 | 조합 K | LogisticRegression |
| KOSPI200 | 조합 C | LogisticRegression |

공식 최종 실행 보고서에도 다음과 같이 기록돼 있습니다.

```json
"models": {
  "index": "C LogisticRegression",
  "stock": "K LogisticRegression"
}
```

## 공식 선정 기준과의 차이

Model Lab과 공식 최종 모델은 두 가지 조건이 다릅니다.

1. 피처 조합이 다릅니다.

   - Model Lab: 조합 E
   - 공식 개별종목: 조합 K
   - 공식 KOSPI200: 조합 C

2. 모델 선정 기준이 다릅니다.

   - Model Lab: Accuracy·Macro F1·하락 Recall 조화평균
   - 공식 개별종목: 학습 최빈 기준선 대비 Accuracy → Macro F1 → 기준선 승리 폴드 수
   - 공식 KOSPI200: 운영 게이트 통과 후 상승 PR-AUC → ΔSharpe → 상승 Precision

## 결론

현재 Model Lab에서 LightGBM이 `BEST MODEL`로 표시되는 것은 공식 최종 모델 선정 결과가 변경됐다는 의미가 아닙니다.

이는 조합 E와 조화평균 기준으로 수행한 별도 실험 결과이며, 발표와 최종 실행에서 확정한 모델은 다음과 같습니다.

- 개별종목: 조합 K + LogisticRegression
- KOSPI200: 조합 C + LogisticRegression

현재 배포 화면에서는 Model Lab의 별도 실험 결과와 공식 최종 모델이 구분되지 않은 상태에서 `BEST MODEL`이라는 명칭을 사용하고 있어, 사용자가 LightGBM을 프로젝트의 공식 최종 모델로 오해할 가능성이 있다는 점을 공유드립니다.

**Comment by rkdalstjr** [comment]
노트북에 E를 가장 좋은 모델로 사용한다는 것만 보고 E로 진행했는데, K와 C로 수정하겠습니다

**Comment by GoonGuMa** [comment]
그러네요 노트북에 혼동을 줄 수 있는 문장이 있는거 확인했습니다
수정해서 PR할게요! 죄송합니다
